paper-autocomplete.html

```diff
@@ -476,7 +485,8 @@
     _fireEvent: function (option, evt) {
       var id=this._getId();
       var event='autocomplete' + this.eventNamespace + evt;
-      this.fire(event,{id:id,value:option.value,text:option.text,target:this,option:option});
+      this.fire(event,{id:id,value:option[this.valueProperty] || option.value,text:option[this.textProperty] || option.text,target:this,option:option
```

Comment:
I will split and format this line to make it more readable. I hope you don't mind.